### DISCUSSION TITLE
视频页面在竖屏时，另一种全屏播放排版（或者默认排版）。

### DISCUSSION THREAD

**Discussion by chenxi-Eumenides** [the1812/Bilibili-Evolved#4180] [功能建议 / Ideas]
### 功能描述
当页面为竖屏（或者较长）时，将整个视频（及其标题和点赞等区域）扩展到整个页面宽度，并固定在页面最上方（类似youtube的剧场模式？）。滚动评论时变动该区域，只滚动评论。有点像平板上的bilibili客户端播放页面。

###  缘由
使用竖屏看视频时，视频的宽屏模式也不能占满整个页面宽度，且评论区域非常适合上方边看视频，下方边浏览评论，等于删去所有的空白区域，整个更紧凑。

### 脚本版本
v2.7.1-49-g261f1b61a

### 附加截图
大概类似与这样？很抱歉很难看清，我只用qq截图随便找了个视频截了一下。

![image](https://user-images.githubusercontent.com/76398980/236247231-d4296c3f-0d0f-45ed-988f-556a72da9207.png)

### 致遇到了相同问题准备回复的人

1、遇到了相同的问题：**请用左下角的 reaction 点赞**
2、实时关注该 discussion 的后续进展：**请用 discussion 页上的 Subscribe 功能**（在 PC 端按钮通常位于页面右侧）。

**Comment by the1812** [comment]
挺有创意, 不过估计会很难写出来...
